moulinette/__init__.py

```diff
@@ -139,3 +138,8 @@ class at construction
         logging.getLogger(namespaces[0]).error(e.strerror)
         return 1
     return 0
+
+
+def env():
```

Comment:
Shouldn't it be "moulinette_env" there then? (and in `__all__`)

I'm not sure to understand the purpose of this function :x

Reply:
For the naming: the rest of the API is quite "short", like "cli", "init", etc.

For the purpose: before you could `from moulinette import LOCALES_DIR` but that was causing problems (explained in first post). So now you can do `from moulinette import env` and then do `env['LOCALES_DIR']`.

Reply:
Oh ok, I guess we'll do some cleaning of that one day ™

Thx for the answer :)